### PR TITLE
Fix 1754: [UI] Make search case insensitive

### LINKED ISSUE #1754: [UI] Make search case insensitive
**Reserved** for [outreachy](https://www.outreachy.org/) candidates. In order for this to be assigned to you, make sure to complete the first step of the application process outlined in the [project description](https://www.outreachy.org/december-2019-to-march-2020-internship-round/communities/mozilla/#build-a-component-that-can-display-a-json-schema).

---

**To Reproduce**
1. Go to https://taskcluster-ui.herokuapp.com/auth/roles/
2. Search for "Github" using the search bar

**Expected behavior**

Screen Shot 2019-10-25 at 5 08 03 PM

**Actual behavior**

Screen Shot 2019-10-25 at 5 08 07 PM

Other views that need this change:
* https://taskcluster-ui.herokuapp.com/auth/clients
* https://taskcluster-ui.herokuapp.com/secrets
* https://taskcluster-ui.herokuapp.com/hooks
* https://taskcluster-ui.herokuapp.com/worker-manager

### PR DESCRIPTION
**Pull request by Karskaya**
Make search case insensitive for Roles and Secrets by adding regexp option \i

Github Bug/Issue: Fixes #1754

### CODE REVIEW

**Comment by Karskaya** [comment]
Hello @helfi92!
I was able to fix Roles and Secrets pages. It was easy.
I need your help, any advice with Clients, Hooks and Worker-manager.

File: changelog/issue-1754.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+level: minor
```
Comment (vishakhanihore) [inline changelog/issue-1754.md]: `Silent` would suit better here

Comment (Karskaya) [reply changelog/issue-1754.md]: @vishakhanihore Oh, ok, thank you! Will change it in next commit

**Comment by Karskaya** [comment]
@helfi92 Thank you for your response. Will start with it tomorrow!

File: changelog/issue-1754.md (hunk 1 above)
Comment (Karskaya) [reply changelog/issue-1754.md]: @vishakhanihore Done!

**Comment by Karskaya** [comment]
Hello, @helfi92! Sorry for delay, please, review Clients and Worker Pools search part.
I'll made a separate PR for Hooks later.

**Comment by helfi92** [comment]
> I'll made a separate PR for Hooks later.

Let me know if you had any questions.

**Comment by Karskaya** [comment]
> This is really good 💯
> 
Thank you!

**Comment by Karskaya** [comment]
> For Hooks, we would need to expose a new prop in https://github.com/helfi92/material-ui-treeview (e.g., `caseInsensitiveSearch`)
> 
> Feel free to do Hooks in a follow-up PR if that's more convenient since it revolves contributing to another library.

Hello @helfi92 !
So... I've forked, cloned material-ui-treeview. Run `yarn`. And then `yarn start`
Errors I see:
`material-ui-treeview Alexa$ yarn start`
`yarn run v1.19.0`
`$ styleguidist server`
`Error: Cannot find module 'babel-plugin-transform-es2015-modules-commonjs'`
`Error: Cannot find module 'babel-plugin-transform-es2015-modules-commonjs'`

Three questions.
1. How I can get rid of this errors? (no help from google or I search in a wrong way)
2. Does it make sense to run material-ui-treeview? What will I see there at http://localhost:5000 ?
The same as this demo https://hassanali.me/material-ui-treeview ? Will I be able to test search?
3. How can I test this in task-cluster? Do I need to test it there at all?

**Comment by helfi92** [comment]
> Error: Cannot find module 'babel-plugin-transform-es2015-modules-commonjs'

Yes, this is a known issue. You can fix that with

```
yarn add -D babel-plugin-transform-es2015-modules-commonjs
yarn add -D babel-plugin-transform-object-rest-spread
```

You should be able to test everything using `yarn start`. You won't need to test that it works in the Taskcluster. `yarn start` will start the styleguide server. Once your pull-request is merged, you can change the Taskcluster UI to incorporate use the new prop.

**Comment by Karskaya** [comment]
Hello @helfi92! Could you please review related PR https://github.com/helfi92/material-ui-treeview/pull/47 ?

**Pull request merged**